mscore/scoretab.cpp

```diff
@@ -42,6 +42,30 @@
 
 namespace Ms {
 
+//---------------------------------------------------------
+//   MsTabBar::mousePressEvent
+//---------------------------------------------------------
+
+void MsTabBar::mousePressEvent(QMouseEvent* e)
+      {
+      QTabBar::mousePressEvent(e);
+      if (e->button() == Qt::MiddleButton)
+            middleClickedTab = tabAt(e->pos());
+      }
+
+//---------------------------------------------------------
+//   MsTabBar::mouseReleaseEvent
+//---------------------------------------------------------
+
+void MsTabBar::mouseReleaseEvent(QMouseEvent* e)
+      {
+      QTabBar::mouseReleaseEvent(e);
+      if (e->button() == Qt::MiddleButton && tabAt(e->pos()) == middleClickedTab) {
+            emit tabCloseRequested(middleClickedTab);
+            middleClickedTab = -1; // reset
```

Comment:
Shouldn't `middleClickedTab` always be reset whenever `e->button() == Qt::MiddleButton` and not just when the tab matches?

In other words:

```
if (e->button() == Qt::MiddleButton) {
      if (tabAt(e->pos()) == middleClickedTab)
            emit tabCloseRequested(middleClickedTab);
      middleClickedTab = -1; // reset
      }
```

Reply:
You're right...